common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -149,6 +149,17 @@ class InheritanceMixin(XBlockMixin):
         default=True,
         scope=Scope.settings
     )
+    video_bumper = Dict(
+        display_name=_("Video Bumper Settings"),
+        help=_(
+            """Enter video bumper settings in the following format:"""
+            """{"""
+            """ "edx_video_id": "edx_video_id","""
```

Comment:
The mobile team and the edX media team have agreed to use only "video ID", without the edX, on the Video Uploads page and in the Video component. (The intention is to minimize rework when this feature is available to the open edX community.) If at all possible, we should be consistent here too, with "video_id".